content/300-guides/200-deployment/201-serverless/150-deploy-to-azure-functions.mdx

```diff
@@ -1,640 +1,42 @@
 ---
 title: 'Deploy to Azure Functions'
-metaTitle: 'How to deploy a REST API to Azure Functions with Azure SQL'
+metaTitle: 'How to deploy an Azure Function using Prisma'
```

Comment:
I've been trying to clarify in our guides that you're not deploying with prisma. You're deploying an app using Prisma to a location.
```suggestion
metaTitle: 'How to deploy an app using Prisma to Azure Functions'
```